src/Meshes/transformNode.ts

```diff
@@ -1078,8 +1096,8 @@ export class TransformNode extends Node {
 
         this._afterComputeWorldMatrix();
 
-        // Absolute position
-        this._absolutePosition.copyFromFloats(this._worldMatrix.m[12], this._worldMatrix.m[13], this._worldMatrix.m[14]);
+        // Update absolute transform values
+        this._worldMatrix.decompose(this._absoluteScaling, this._absoluteRotationQuaternion, this._absolutePosition);

```

Comment:
That is an expensive function we cannot call in the compute world matrix function (which can be called thousand of times per frame.

I would recommend having a flag in the computeWM function that indicates that absolute scaling and rotation are out of sync, then in the getters check that flag and update the _absoluteScaling and _absoluteortationquaternion before returning them

Reply:
Updated!